src/daemon/http/server.rs

```diff
@@ -558,10 +563,52 @@ pub async fn metrics(req: Request) -> RoutingResult {
                 }
             }
 
+            {
+                // CA -> Repository status
+
+                res.push('\n');
+                res.push_str("# HELP krill_ca_repo_success status of last ca to publication server connection (0=issue, 1=success)\n");
+                res.push_str("# TYPE krill_ca_repo_success gauge\n");
+                for (ca, status) in ca_repo_status.iter() {
+                    // skip the ones for which we have no status yet, i.e it was really only just added
+                    // and no attempt to connect has yet been made.
+                    if let Some(exchange) = status.last_exchange() {
+                        let value = if exchange.was_success() { 1 } else { 0 };
+                        res.push_str(&format!("krill_ca_repo_success{{ca=\"{}\"}} {}\n", ca, value));
+                    }
+                }
+
+                res.push('\n');
+                res.push_str("# HELP krill_ca_repo_last_success_time timestamp of last successful ca to publication server connection\n");
+                res.push_str("# TYPE krill_ca_repo_last_success_time gauge\n");
+                for (ca, status) in ca_repo_status.iter() {
+                    // skip the ones for which we have no status yet, i.e it was really only just added
+                    // and no attempt to connect has yet been made.
+                    if let Some(last_success) = status.last_success() {
+                        res.push_str(&format!(
+                            "krill_ca_repo_last_success_time{{ca=\"{}\"}} {}\n",
+                            ca,
+                            last_success.timestamp()
+                        ));
+                    }
+                }
+
+                res.push('\n');
+                res.push_str("# HELP krill_ca_repo_next_before_time timestamp of next planned ca to publication server connection\n");
```

Comment:
Consistency: Sometimes "ca" is mentioned in metric help texts and other times "CA" is mentioned.

Reply:
fixed (on the branch this was merged into)